src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2529,6 +2529,7 @@ private void PackagesMenuGuide_Click(object sender, RoutedEventArgs e)
             try
             {
                 dynamoViewModel.MainGuideManager.LaunchTour(GuidesManager.PackagesGuideName);
+                dynamoViewModel.OnEnableShortcutBarItems(false);
```

Comment:
Just like how you implemented above, can you put this call as part of `LaunchTour` call?

Reply:
Done.